tools/config-generator/main.go

```diff
@@ -682,37 +495,6 @@ func outputConfig(line string) {
 	}
 }
 
-// strExists checks if the given string exists in the array
-func strExists(arr []string, str string) bool {
-	for _, s := range arr {
-		if str == s {
-			return true
-		}
-	}
-	return false
-}
-
-type specialBranchLogic struct {
```

Comment:
It looks like this went away entirely?

Reply:
Yeah, these functions are not used anymore.